src/prometheus/prometheus-config.yaml

```diff
@@ -0,0 +1,10 @@
+global:
+  evaluation_interval: 30s
+  scrape_interval: 30s
```

Comment:
Do we think 30s scraping interval is too long for the demo? Maybe 5s to make it more interactive?

Reply:
I changed to 5s.